packages/hurl/src/http/error.rs

```diff
@@ -70,6 +71,7 @@ impl HttpError {
             HttpError::InvalidUrl(..) => "Invalid URL".to_string(),
             HttpError::Libcurl { .. } => "HTTP connection".to_string(),
             HttpError::LibcurlUnknownOption { .. } => "HTTP connection".to_string(),
+            HttpError::NoPrimaryIpInResponse => "HTTP connection".to_string(),
```

Comment:
Another change, just `NoPrimaryIpResponse` to `NoPrimaryIp`, I prefer it to be more consise